From 78a01bfa242139ae6b7ac487c0e857457d8ff416 Mon Sep 17 00:00:00 2001
From: Mikhail Kshevetskiy <mikhail.kshevetskiy@iopsys.eu>
Date: Mon, 13 Oct 2025 20:56:31 +0300
Subject: [PATCH 24/24] arm: airoha: enable position independent code

This slightly increase the code, but makes debugging a bit easy

Signed-off-by: Mikhail Kshevetskiy <mikhail.kshevetskiy@iopsys.eu>
---
 configs/an7581_evb_defconfig | 1 +
 configs/an7583_evb_defconfig | 1 +
 configs/en7523_evb_defconfig | 1 +
 3 files changed, 3 insertions(+)

diff --git a/configs/an7581_evb_defconfig b/configs/an7581_evb_defconfig
index afdb0cd8586..be076ec7723 100644
--- a/configs/an7581_evb_defconfig
+++ b/configs/an7581_evb_defconfig
@@ -1,5 +1,6 @@
 CONFIG_ARM=y
 CONFIG_ARCH_AIROHA=y
+CONFIG_POSITION_INDEPENDENT=y
 CONFIG_TARGET_AN7581=y
 CONFIG_TEXT_BASE=0x81E00000
 CONFIG_SYS_MALLOC_F_LEN=0x4000
diff --git a/configs/an7583_evb_defconfig b/configs/an7583_evb_defconfig
index c3d47c411ba..7ef2d6feeba 100644
--- a/configs/an7583_evb_defconfig
+++ b/configs/an7583_evb_defconfig
@@ -1,5 +1,6 @@
 CONFIG_ARM=y
 CONFIG_ARCH_AIROHA=y
+CONFIG_POSITION_INDEPENDENT=y
 CONFIG_TARGET_AN7583=y
 CONFIG_TEXT_BASE=0x81E00000
 CONFIG_SYS_MALLOC_F_LEN=0x4000
diff --git a/configs/en7523_evb_defconfig b/configs/en7523_evb_defconfig
index 8febb6cabdd..53011e47f55 100644
--- a/configs/en7523_evb_defconfig
+++ b/configs/en7523_evb_defconfig
@@ -1,6 +1,7 @@
 CONFIG_ARM=y
 CONFIG_SYS_ARCH_TIMER=y
 CONFIG_ARCH_AIROHA=y
+CONFIG_POSITION_INDEPENDENT=y
 CONFIG_TEXT_BASE=0x81E00000
 CONFIG_SYS_MALLOC_F_LEN=0x4000
 CONFIG_NR_DRAM_BANKS=1
-- 
2.51.0

